client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -324,6 +328,14 @@ export class CommentNavigation extends Component {
 	}
 }
 
+const filterBulkActions = function( value, key ) {
+	return key.indexOf( COMMENTS_CHANGE_STATUS ) !== -1 || key.indexOf( COMMENTS_DELETE ) !== -1;
+};
```

Comment:
are you sure this shouldn't just be `key === COMMENTS_CHANGE_STATUS || key === COMMENTS_DELETE`

Reply:
key here is the full serialized action object (minus meta)

Reply:
oh I see - you are doing a string search, not a list search…